Overview.src.html

```diff
@@ -805,6 +823,15 @@ <h4 id=terminology-headers>Headers</h4>
 <span data-anolis-ref>REFERRER</span>
 
 <p>A <span title=concept-request>request</span> has an associated
+<dfn title=concept-request-client-hints-list>client hints list</dfn>, which is a
+<span title=concept-client-hints-list>client-hints list</span>. Unless stated otherwise, it is
+the empty list.
+
+<p class="note no-backref">This can be used to override a client hints list associated with
+an <span data-anolis-spec=html>environment settings object</span>.
+<span data-anolis-ref>CLIENT-HINTS</span>
```

Comment:
CLIENT-HINTS doesn't define a client hints list associated with an environment settings object. What exactly is the purpose of this?